### ISSUE TITLE
Change labels and column order in Master lists manager

### ISSUE THREAD

**Issue by SteeveEbener** [terraframe/geoprism-registry#81]
Applies to when looking at the lists extracted for a given GeoObject type.

Different elements to be modified when you reached the list of master lists extracted from the graph for a given object:

1. Replace "Publishing for:" by "Validity date:"
2. Replace "Publish New Version" on the button by "Extract master list"
3. Change the order and label of the columns in the table to be consistent with the other windows as follow:
    - "For Date" with label to be replaced by "Validity"
    - "Create Date" with label to be replaced by "Extraction date"

**Comment by dtreatman** [comment]
I agree with the replacements with the term "validity", this makes sense, but I am wary that the "Extract master list" and "Extraction date" are not very intuitive for regular users. However, I think this is an important label to get right early on.

I suggest we keep this in Sprint 0.7, but request that we run a few options by CHAI and get feedback on what makes sense to them.

**Comment by SteeveEbener** [comment]
Apart from being interested in knowing why this issue has been removed from the sprint 0.7, I think this is something important to be addressed in relation to forthcoming demos.

This being said, I am now in favor of:

- replacing "Publishing for" by "Extract for" and "Publish New version" by "Extract new Version" on the button.

- Keeping "For Date" in the table but still replacing "Create Date" by "Extraction date"

**Comment by dtreatman** [comment]
I agree that the labels are not currently intuitive and propose that we put
forward a few different options to users and see what makes the most sense
to them. Can we think of any other terms that we might be able to test to
see what is more intuitive to users? Starting a list here:

Meaning: produce a list that contains the set of GeoObjects and attributes
that were valid for a specified date

Label options:

   - Publishing for / Publish new version
   - Extract for / Extract new version
   - Print for / Print new version
   - Create list valid for / Create new version

Any other options?

**Comment by SteeveEbener** [comment]
Thanks Derek.

According to me the following two options should be removed from this list because they generate confusion:
- print because we are not generated a paper output
- publish because we need to keep this concept for what is being shared with users outside the organization having the mandate over the data + we have the published section in the list module

Between the remaining, I continue to go for "Extract" because the data do exist in the graph 
 while "Create" gives the impression that the CGR is generating something from "scratch"

We need to ensure that the front end contributes to explaining to user how the back end works to avoid for the latter to become a "black box"

**Comment by CarlyNel** [comment]
Update: We've put together some [visual mockups](https://docs.google.com/presentation/d/11vpH3W6FqQ4lIXx_pco8_fcMzO14GpbQ/edit#slide=id.p3) and will be reaching out to CHAI for their feedback and label preferences. I'll share their feedback on this ticket once received.

**Comment by SteeveEbener** [comment]
Thanks Carly

Just one important point for clarification: this ticket has been generated in relation to what is now the "exploratory" section of the list module.

=> these changes should be discussed in relation to that section and not the "published" one (I am not even sure that the "published" section was actually existing at the time of generating the ticket).

=> According to me, nothing should  be changed in the "published" section because it indeed refers to the instances of the lists which are published (made accessible to) the other organization or the anonymous user when the data is made public.

The concept of "publishing" is the one also used when defining the frequency at which (and normally also soon the data from which) these instances are being published when creating a master list.

In conclusion, the following should normally be removed from the deck:
- Slide 1 
- the question regarding the "published" section in slide 3

Regarding slide 2,  this depends if this is a list that has been "published" or "extracted"

**Comment by dtreatman** [comment]
Steeve, thanks for the clarification, I also misunderstood your suggestion.
Making these changes to the terms for Exploratory lists only makes sense.
Agree with your suggested changes to the mockups.

**Comment by SteeveEbener** [comment]
No problem. Justin confirmed that the original ticket was issued before the creation of the "published" section which contributed to the confusion.

Thanks in advance for he adjustment of the mockups and take care

**Comment by CarlyNel** [comment]
Sent a reminder via Slack yesterday to Anne, Pedro, Venky and Sameen. Anne acknowledged my request and said that her team will review and revert.

**Comment by CarlyNel** [comment]
Please find the mockups for the label changes [here](https://docs.google.com/presentation/d/11vpH3W6FqQ4lIXx_pco8_fcMzO14GpbQ/edit#slide=id.g88b3b82627_0_27) which now incorporate CHAI's feedback.

**Comment by CarlyNel** [comment]
I have incorporated all of Steeve's feedback on [these label changes](https://docs.google.com/presentation/d/11vpH3W6FqQ4lIXx_pco8_fcMzO14GpbQ/edit#slide=id.g88b3b82627_0_1) so we should now be good to go.

**Comment by justinlewis** [comment]
@CarlyNel & @steeveebener

I've done all these changes (released in 0.8.1) with two notes/exceptions:
1. I added the help text (sentence) on the publish versions button as a title attribute on the button so it only shows up when you hover over the button. I did this because the UI is getting all cluttered and I didsn't see how to add a sentence in a good way. 
2. We can't add a refresh button to the data view of the published list because of the level of effort to support that. We can consider this in the future but the current development effort is tight and this is more challenging for published lists than it seems.

**Comment by SteeveEbener** [comment]
@Justin. That's what I was expecting for 1.

2, was actually a question from my side more than a request in the first place. Does it actually make sense to have a refresh button there? 
 
Normally, only the RA/RM of the organization having the curation mandate over the list should be able to publish or republish (refresh) the lists. The users from any other organization should automatically have access to the most recently republished (refreshed) version of the list.

Now that I am thinking about it, organization B should not even see the "publish list" button.

**Comment by justinlewis** [comment]
It sounds like this is satisfied then? Please close this ticket if you agree.

**Comment by SteeveEbener** [comment]
@justinlewis  You are right :) Closing
